plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -166,4 +171,56 @@ public IcebergConfig setProjectionPushdownEnabled(boolean projectionPushdownEnab
         this.projectionPushdownEnabled = projectionPushdownEnabled;
         return this;
     }
+
+    @NotNull
+    public DataSize getSplitTargetSize()
+    {
+        return splitTargetSize;
+    }
+
+    @Config("iceberg.split-target-size")
+    public IcebergConfig setSplitTargetSize(DataSize splitTargetSize)
+    {
+        this.splitTargetSize = splitTargetSize;
+        return this;
+    }
+
+    @NotNull
+    public DataSize getSplitMetadataTargetSize()
+    {
+        return splitMetadataTargetSize;
+    }
+
+    @Config("iceberg.split-metadata-target-size")
+    public IcebergConfig setSplitMetadataTargetSize(DataSize splitMetadataTargetSize)
```

Comment:
What is the effect of setting this property?

Reply:
@findepi looks like by Iceberg while scanning metadata for metadata tables like `$manifests` [BaseMetadataTableScan.java](https://github.com/apache/iceberg/blob/97055c4114d396eeef24d8280006984b1c2cb306/core/src/main/java/org/apache/iceberg/BaseMetadataTableScan.java#L37-L38) 

As Trino doesn't use the Metadata table APIs of Iceberg, so maybe this property is not required for processing the regular queries.

what do you think @fengguangyuan

Reply:
@findepi Metadata file  is AVRO format, which is also splittable.

@osscm Got it. I will delete these unused code today.